src/server/controllers/UserController.js

```diff
@@ -126,4 +126,24 @@ async function getUserfromToken(token) {
     return null;
 }
 
-module.exports = { registerUser, loginUser, tokenChecker };
\ No newline at end of file
+const getUserBio = async (req, res) => {
+    try {
+        const { userId } = req.body;
+        const usersCollection = await dbClient.getCollection('chatDB', 'users')
+        const user = await usersCollection.findOne({_id: userId});
```

Comment:
I think searching with the raw string userId wont work convert it to an objectId

Reply:
ooh, okay, i will change that